jupyter-docker/run-jupyter-local.sh

```diff
@@ -12,8 +12,7 @@ start () {
     fi
 
     echo "Starting Jupyter server container..."
-    docker create -it --name ${CONTAINER} --rm -p 8001:8001 $DOCKER_IMG 'export PYTHONPATH=$PYTHONPATH:/etc/jupyter/custom;start-notebook.sh'
-    docker cp jupyter-docker/jupyter_notebook_config.py ${CONTAINER}:/etc/jupyter/jupyter_notebook_config.py
```

Comment:
oh, it's added to the dockerfile

Reply:
`jupyter-docker/jupyter_notebook_config.py` doesn't really work with `run-jupyter-local.sh` because it relies on some EVs, etc that are set by Leo and not in `jupyter/minimal-notebook`. This script just starts Jupyter in a docker with default configs (no Leo needed). I use it for testing Jupyter-specific stuff like extensions without needing to run a Leo/dataproc.

Reply:
I think @vkumra-broad spent some time using it to test the notebook extension, so we should at least make its intended usage clear.